### PR TITLE
README Update from Customer Feedback

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Comment by prb112** [comment]
Thanks to E. Babenko/IBM for the feedback and edits.

File: fhir-persistence-schema/README.md [hunk 1]
```diff
@@ -214,7 +214,55 @@ For postgresql:
 --db-type postgresql
 ```
 
-## Alternative: manually apply the schema
+## Alternative: Setting upIBM FHIR Servers against a shared DB2 with separate schemas for each customer
```
Comment (lmsurpre) [inline fhir-persistence-schema/README.md:217]:
```suggestion
## Alternative: Setting up a shared Db2 with separate schemas for each tenant
```

File: fhir-persistence-schema/README.md [hunk 2]
```diff
@@ -214,7 +214,55 @@ For postgresql:
...
+```
+
+```
+java -jar ./fhir-persistence-schema-${VERSION}-cli.jar \ 
+--prop-file db2.properties
+--grant-to FHIRSERVER
+--target DATA FHIRDATA_2ND
+```
+
+## Alternative: Print the Schema
```
Comment (lmsurpre) [inline fhir-persistence-schema/README.md:265]: We already have a section on printing the schema near the top (under getting started).  This section should specifically focus on how to apply that manually.

```suggestion
## Alternative: Manually apply the schema
```

**Comment by prb112** [comment]
fixed

**Pull request merged**